.github/workflows/docs.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build and Deploy Documentation
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  create_documentation:
+    runs-on: ubuntu-latest
+    name: Deploy documentation
+
+    steps:
+
+    - name: Install ubuntu dependencies
+      run: |
+        echo "Install ubuntu dependencies (if any)"
+
+    - name: Install (upgrade) dependencies
+      run: |
+        pip install --upgrade pip
+        pip install pdoc
```

Comment:
A PR would be great for `requirements.txt` and `environment.yml`.

Reply:
you can installl packages via pip in conda.

Reply:
That's right, but you still would do that from the environment.yml (or import the `requirements.txt`), but it seems that that is for managing pip packages that are not available from conda. So, some packages would come from conda, others from pip, but there wouldn't be an option on how a given package is retrieved/installed.

Reply:
A PR would be great for `requirements.txt` and `environment.yml`
`pip` does not require `requirements.txt` when the `setup.cfg` and `setup.py` are present.
Installation does not require `pip`, though it is made convenient.
With those setup files, one can install as follows:
```
python setup.py install
```

Reply:
I asked another team member about this:

> We had discussed also supporting conda environments which would mean there is duplication of this canonical information since we'd need a requirements.txt for pip installs, and an environment.yml for conda support and both include the same set of packages, but those should be the only two places where we maintain the list of packages needed for this repo.  It sure would be nice if anybody here knows a way to maintain both those files from a single source of information....

I am not very familiar with all of this, but this is what she said:

>  You can generate a requirements.txt file indirectly from your conda environment, but you have to activate your conda environment and then you generate both the environment.yml and requirements.txt file:
    conda env export>environment.yml
    pip list --format=freeze>requirements.txt
I do this to create the requirements.txt files required for my binder account (for jupyter notebook sharing).
Sometimes conda won't let you install some things and you have to use pip within your conda environment to install and sometimes those packages don't get picked up in the environments.yml file when you export your conda env.

I'm not sure if that is helpful or not, but wanted to pass it along.

Reply:
Thanks for chiming in on this as well.
`pip freeze` and `conda env export` are after the installation commands that are useful to share with the user or community the runtime environment.  The files produced from these commands can then be used to create the environment itself.

At this point in this empty project, doing these commands will return more or less an empty list as we don't have any code in this python module.

The discussion here has been centered around what is the level of support this package developers need to provide.
From the `MetPlus` repository, I see they also have `setup.py`, `requirements.txt` and `environment.yml`. We can follow the same path if that is desired by the team.

Reply:
Follow up - isn't this discussion basically UW-37 "As a code maintainer, I'd like to provide an environment.yaml or wheels file so that it's easier to port our Python environment." ? If so, we are allowed to ignore this until sprint 3. ;)

Reply:
I concur with @benjamin-cash that we could accept this as a good initial commit and make sure to address the items above in a separate ticket.

Reply:
It looks like UW-35 might be relevant to this discussion too.  I'm going to mark this as resolved in the hopes that we can address this feedback in future tickets planned for the sprint.